AppKit/CPPopUpButton.j

```diff
@@ -685,14 +692,16 @@ CPPopUpButtonStatePullsDown = CPThemeState("pulls-down");
     if ([self pullsDown])
     {
         var positionedItem = nil,
-            location = CGPointMake(0.0, CGRectGetMaxY(bounds) - 1);
+            menuOffset     = [self currentValueForThemeAttribute:@"menu-offset"],
+            location       = CGPointMake(menuOffset.width, CGRectGetMaxY(bounds) + menuOffset.height);
     }
     else
     {
         var contentRect = [self contentRectForBounds:bounds],
             positionedItem = [self selectedItem],
             standardLeftMargin = [_CPMenuWindow _standardLeftMargin] + [_CPMenuItemStandardView _standardLeftMargin],
-            location = CGPointMake(CGRectGetMinX(contentRect) - standardLeftMargin, 0.0);
+            menuOffset = [self currentValueForThemeAttribute:@"menu-offset"],
+            location = CGPointMake(CGRectGetMinX(contentRect) - standardLeftMargin + menuOffset.width, 0.0 + menuOffset.height);
```

Comment:
If I understand this correctly so will this put the location one pixel to the left compared to the previous behavior? Is this to correct something a position of something? Just so we understand why this is done.

Reply:
Well, indeed the default value of ```menu-offset``` should be (0, 0).
And also, this ```0.0 + ...``` is hurting my eyes !
;-)

BTW, just to answer your question, with the previous default value, it would put the location one pixel upper (as default value was (0, -1))